tests/pytests/test_adcc.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+
+from utils import *
+
+import psi4
+import numpy as np
+from pathlib import Path
+import json
+from addons import using
+
+
+pytestmark = [pytest.mark.psi, pytest.mark.api]
+
+
+with open(Path(__file__).parent / "adcc_reference_data.json") as f:
+    reference_data = json.load(f)
```

Comment:
I used to have this behind a fixture, but wasn't sure how to get it working with l.19-33 below...

Reply:
good point -- I don't know how to deal with that either. best to leave as-is for now, I suppose. at least reference_data gets acted upon at collection time rather than changing during runtime.

Reply:
I think there's no `pytest` feature right now that allows one to handle this cleanly. Let's leave it as is for now 😄